src/pollers/mod.rs

```diff
@@ -113,26 +113,35 @@ pub trait ServerGatewayApis {
     async fn poll_activity_task(&self, task_queue: String) -> Result<PollTaskResponse>;
 
     /// Complete a task by sending it to the server. `task_token` is the task token that would've
-    /// been received from [PollWorkflowTaskQueueApi::poll]. `commands` is a list of new commands
+    /// been received from [poll_workflow_task_queue] API. `commands` is a list of new commands
```

Comment:
Should just be `[poll_workflow_task]`

Reply:
Looks like the other added comments have the same extra `_queue`

Reply:
It's the temporal API name, so I though we want to use it as is to avoid confusion.